libs/designsystem/src/lib/components/modal/services/modal.helper.spec.ts

```diff
@@ -230,6 +244,7 @@ describe('ModalHelper', () => {
             expectShadowStyle();
             expectBackdropStyle();
             expectDrawerWrapperStyle();
+            expectBodyToAllowScroll();
```

Comment:
Probably better to move the test regarding `interactWithBackground`  into it's own `describe` block and not modify the regular drawer tests

Reply:
Changed in latest commit